app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,10 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = Comment.create(author_id: current_user.id, post_id: params[:post_id], text: params[:comment][:text])
+    if @comment.save
+      redirect_to user_post_path(params[:user_id], params[:post_id]), notice: 'Comment was successfully created.'
+    else
+      render :new
+    end
+  end
+end
```

Comment:
Good job working on the project requirements. But unfortunately you have failing tests it would be nice if you fixed this issue.

Take a look here :
![Screenshot 2023-06-23 at 4 13 11 PM](https://github.com/Ridwanullahi-code/BlogApp/assets/79506105/c617770e-c255-4663-94d1-21ab70bcffaa)